pom.xml

```diff
@@ -503,37 +508,6 @@
                     <version>1.3</version>
                 </plugin>
                 <plugin>
-                    <groupId>org.bsc.maven</groupId>
```

Comment:
What's the reason for removing this? I liked the fact that this set-up does not add the processor as normal project dependency but only on the separate processor path. There is a separate artifact with only the annotations which can be used as project dependency.